web3_proxy/src/rpcs/blockchain.rs

```diff
@@ -72,16 +96,16 @@ impl Web3ProxyBlock {
     }
 
     pub fn age(&self) -> u64 {
-        let now = SystemTime::now()
-            .duration_since(UNIX_EPOCH)
-            .expect("there should always be time");
+        let now = chrono::Utc::now().timestamp();
 
-        let block_timestamp = Duration::from_secs(self.block.timestamp.as_u64());
+        let block_timestamp = self.block.timestamp.as_u32() as i64;
 
         if block_timestamp < now {
             // this server is still syncing from too far away to serve requests
             // u64 is safe because ew checked equality above
-            (now - block_timestamp).as_secs()
+            // (now - block_timestamp).as_secs()
```

Comment:
delete this stale code

Reply:
In a latest merge it looks like this snippet is used:
`(now - block_timestamp) as u64`. 
Does this expression here return seconds? `(now - block_timestamp)`

Reply:
resolved